hieradata/role.ci-slave.3.yaml

```diff
@@ -1,5 +1,19 @@
 ---
 ci_environment::jenkins_slave::labels:
   - 'elasticsearch-1.4'
-  - 'mongodb-2.4'
+  - 'mongodb-3.2'
   - 'postgresql-9.3'
+
+apt::sources:
+  'mongodb3.2':
+    location: 'http://repo.mongodb.org/apt/ubuntu/'
```

Comment:
Yes.  I could not get apt::sources to use apt.publishing.service.gov.uk when defining in hiera.  Always 404: http://apt.publishing.service.gov.uk/mongodb3.2/dists/trusty-mongodb-org/3.2/multiverse/binary-amd64/Packages